daffodil-runtime1/src/main/scala/org/apache/daffodil/processors/EvEncoding.scala

```diff
@@ -93,10 +93,11 @@ abstract class CharsetEvBase(encodingEv: EncodingEvBase, tci: DPathCompileInfo)
 
   override def compute(state: ParseOrUnparseState) = {
     val encString = encodingEv.evaluate(state)
-    val cs = CharsetUtils.getCharset(encString)
```

Comment:
Should this still use `CharsetUtils`, and `CharsetUtils` should be the only thing that uses the registry? This way we can easily change how the registry works if we ever need to since it's hidden behind `CharsetUtils`

Reply:
This comment wasn't addressed.

Reply:
reverted